bindings/web5_uniffi/src/web5.udl

```diff
@@ -42,28 +41,28 @@ enum Dsa {
   "Ed25519"
 };
 
-[Trait]
+[Trait, WithForeign]
 interface Signer {
   [Throws=RustCoreError]
-  bytes sign([ByRef] sequence<u8> payload);
+  bytes sign(bytes payload);
```

Comment:
Oh nice, and @nitro-neal this improves things too, because no longer do we need to use `UByteArray` and instead can just use `ByteArray` (discovered this because [I'm reworking tbdex-rs](https://github.com/TBD54566975/tbdex-rs/pull/64) to also have these improvements)

Reply:
(So when we run `just bind` next time, this will probably break things, because we'll have to rework the types)

Reply:
Awesome!